src/server/pkg/testutil/cmds.go

```diff
@@ -93,9 +87,23 @@ func BashCmd(cmd string, subs ...string) *TestCmd {
 	// Warn users that they must install 'match' if they want to run tests with
 	// this library, and enable 'pipefail' so that if any 'match' in a chain
 	// fails, the whole command fails.
+	// Note that to avoid SIGPIPE errors from derailing entire scripts, we
```

Comment:
this seems to end abruptly

Reply:
Meant to remove this line along with the ones that used to be below it :-).